CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 3.0.0
+
+- Contains function has been renamed to ContainsPath
+- ContainsField, ContainsValue and Reverse functions added
```

Comment:
I would say to rephrase to "Rename Contains function to ContainsPath" and "Add ContainsField, ContainsValue and Reverse functions".

Reply:
The documentation uses this wording:

```
* Added an alias of the Contains function called ContainsPath, and deprecated the Contains function.

* Added the new functions ContainsField and ContainsValue functions, to make it easier to explore the structure of objects and documents.
```